app/controllers/report_templates_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ReportTemplatesController < TemplatesController
+  include Foreman::Controller::Parameters::ReportTemplate
+  helper_method :documentation_anchor
+
+  def documentation_anchor
+    '4.11ReportTemplates'
+  end
+
+  def generate
+    find_resource
```

Comment:
Nit: I'd use a before_action for this.

Reply:
I wanted too, but it's inherited from templates_controller, if I specify it here, I'd need to copy all listed actions from templaters_controller, adding just generate action here overrides the whole list. I could extract the list of action to some constant/method but this felt slightly more straightforward.